solution/ui/regulations/css/scss/partials/_results_item.scss

```diff
@@ -0,0 +1,129 @@
+//-----------------------------
+// Results Item
+//-----------------------------
+
```

Comment:
this should be an omnibus partial that contains all styles for a results item and its constituent components.  These styles had been distributed over several SCSS stylesheets and Single File Components as CSS in JS.